solanches/models.py

```diff
@@ -235,6 +251,29 @@ def to_dict(self):
         cardapio = vars(self).copy()
         return cardapio
 
+    @staticmethod
+    def add_categoria(cardapio_id, categoria):
+        query = {"_id": cardapio_id}
+        cardapio = Cardapio.get_by_id(cardapio_id)
+        new_categoria = cardapio.get("categorias")
+        new_categoria += categoria if type(categoria) is list else [categoria]
+        new_values = {"$set": {"categorias": new_categoria}}
+        DB.cardapio.update_one(query, new_values)
```

Comment:
Pode remover a verificação do `is list`, porque agora vamos adicionar apenas uma categoria mesmo por vez
```suggestion
        categorias = cardapio.get("categorias")
        categorias.append(categoria)
        new_values = {"$set": {"categorias": categorias}}
        DB.cardapio.update_one(query, new_values)
```